src/main/kotlin/me/zeroeightsix/kami/feature/module/AutoGG.kt

```diff
@@ -0,0 +1,54 @@
+package me.zeroeightsix.kami.feature.module
+
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.event.PacketEvent
+import me.zeroeightsix.kami.gui.text.CompiledText
+import net.minecraft.network.packet.s2c.play.GameMessageS2CPacket
+
+@Module.Info(
+    name = "AutoGG",
+    category = Module.Category.MISC,
+    description = "Says GG or a custom message at the end of a match"
+)
+object AutoGG : Module() {
+    @Setting
+    var ggMessage = CompiledText(mutableListOf())
+
+    private const val updateLimit: Long = 2000L // 2 Seconds
```

Comment:
No need to explicitly declare long here, if you have the L. I'm pretty sure Intellij should warn you about this